nixos/modules/services/web-apps/isso.nix

```diff
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.isso;
+
+  inherit (lib) mkEnableOption mkIf mkOption types;
+in {
+
+  options = {
+    services.isso = {
+      enable = mkEnableOption "Whether to enable isso, a commenting server similar to Disqus.";
+
+      user = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service user.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service group.";
+      };
+
+      config = mkOption {
+        description = "Isso configuration file content.";
+        type = types.lines;
+        default = "";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.etc."isso.conf".text = ''
+      ${cfg.config}
+    '';
+
+    systemd.services.isso = {
+      description = "isso, a commenting server similar to Disqus";
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        User = cfg.user;
+        Group = cfg.group;
+
+        DynamicUser = true;
+        ExecStart = ''
+          ${pkgs.isso}/bin/isso
```

Comment:
We should write down in some docs this is good enough for small deployments and real deployments might require a real server, e.g. uWSGI/etc.